index.html

```diff
@@ -13461,6 +13453,69 @@ <h2>Including Elements in the Accessibility Tree</h2>
             </li>
           </ul>
 	</section>
+	<section id="tree_relationships">
+	  <h2>Relationships in the Accessibility Tree</h2>
+          <p>The following terms are used to describe relationships between <abbr title="Document Object Model">DOM</abbr> elements.</p>
+          <p>The <dfn data-export="" data-lt="accessibility child|owned child|child element|child|children|child elements">accessibility children</dfn> of a DOM element are all of the following (excluding those DOM elements that have no corrosponding <a>accessible object</a> because they have been <a href="#tree_exclusion">excluded from the accessibility tree<a>, and excluding any elements whose corrosponding <a>accessible object</a> have been reparented in the <a>accessibility tree</a> via <pref>aria-owns</pref>):</p>
+          <ul>
+            <li>The <abbr title="Document Object Model">DOM</abbr> children of the <a>element</a>.
+            </li>
+            <li>All descendant of the <a>element</a> with only elements of role <rref>generic</rref> or <rref>none</rref> intervening.
+            </li>
+            <li>All children specified via an <pref>aria-owns</pref> relationship to the element.
+            </li>
+            <li>All descendants of an element with role <rref>generic</rref> or <rref>none</rref> specified via <pref>aria-owns</pref> with only elements of role <rref>generic</rref> or <rref>none</rref> intervening.
+            </li>
+          </ul>
+          <p>In the following example, the <rref>list</rref> element has four accessibility children:</p>
+	  <pre class="example highlight">
+&lt;div role="list" aria-owns="child3 child4"&gt;
+  &lt;div role="listitem"&gt;Accessibility Child 1&lt;/div&gt;
+  &lt;div&gt;
+    &lt;div role="listitem"&gt;Accessibility Child 2&lt;/div&gt;
+  &lt;/div&gt;
+&lt;/div&gt;
+&lt;div id="child3" role="listitem"&gt;Accessibility Child 3&lt;/div&gt;
+&lt;div id="child4"&gt;
+  &lt;div role="listitem"&gt;Accessibility Child 4&lt;/div&gt;
+&lt;/div&gt;
```

Comment:
```suggestion
  &lt;/div&gt;
  &lt;div role="listitem" aria-hidden="true"&gt;Excluded element&lt;/div&gt;
  &lt;div id="reparented"&gt;Reparented element&lt;/div&gt;
&lt;/div&gt;
&lt;div id="child3" role="listitem"&gt;Accessibility Child 3&lt;/div&gt;
&lt;div id="child4"&gt;
  &lt;div role="listitem"&gt;Accessibility Child 4&lt;/div&gt;
&lt;/div&gt;
&lt;div aria-owns="reparented"&gt;&lt;/div&gt;
```

At the risk of overcomplicating this example, what would you think about including the two exclusion cases to hammer home that the `list` still only has four accessibility children?

Here’s a quick screencap of my local edit to hopefully make my suggestion more readable 😅 :

<img width="567" alt="image" src="https://github.com/w3c/aria/assets/1395630/206ed5b3-9323-4e0e-824a-0c0b055acd8e">

Reply:
Oooo interesting I was thinking about whether or not to somehow include the excluded cases... Maybe we should have a second example like this, prefaced by `In the following example, the list element has no accessibility children`

```
<div role="list">
  <div role="listitem" aria-hidden="true">Excluded element</div>
  <div role="listitem" id="reparented">Reparented element</div>
</div>
<div aria-owns="reparented"></div>
```

Reply:
Sounds great 👌🏻